var/spack/repos/builtin/packages/psblas/package.py

```diff
@@ -0,0 +1,211 @@
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Psblas(AutotoolsPackage):
+    """PSBLAS: Parallel Sparse BLAS. A library of distributed sparse
+    linear algebra with support for GPU and Multithread acceleration.
+    Part of the PSCToolkit: Parallel Sparse Computation Toolkit."""
+
+    # Url for your package's homepage here.
+    homepage = "https://psctoolkit.github.io/"
+    git = "https://github.com/sfilippone/psblas3"
+
+    # List of GitHub accounts to notify when the package is updated.
+    maintainers("cirdans-home", "sfilippone")
+
+    # SPDX identifier of the project's license below.
+    # See https://spdx.org/licenses/ for a list. Upon manually verifying
+    # the license, set checked_by to your Github username.
+    license("BSD-3-Clause", checked_by="cirdans-home")
+
+    version("development", branch="development")
+    version(
+        "3.9.0-rc1",
+        sha256="7a7091ce52582b6fc442e8793e36461be36c0947272ea803ad72736ec2d56da8",
+        preferred=True,
+        url="https://github.com/sfilippone/psblas3/archive/refs/tags/v3.9.0-rc1.tar.gz",
+    )
+    version(
+        "3.8.1-2",
+        sha256="285ddb7c9a793ea7ecb428d68cf23f4cc04f1c567631aa84bc2bedb65a3d1b0c",
+        url="https://github.com/sfilippone/psblas3/archive/refs/tags/v3.8.1-2.tar.gz",
+    )
+    version(
+        "3.8.1-rc1-3",
+        sha256="00b994158d12644853180d1115ed530b1aed3903c38a9d7bc3d3efc203b2890b",
+        url="https://github.com/sfilippone/psblas3/archive/refs/tags/v3.8.1-rc1-3.tar.gz",
+        deprecated=True,
+    )
+    version(
+        "3.8.1-rc1-1",
+        sha256="4f7371000592d73603010adef654d4e9d27a9692d5204db20c0f7b39c462fd83",
+        url="https://github.com/sfilippone/psblas3/archive/refs/tags/v3.8.1-rc1-1.tar.gz",
+        deprecated=True,
+    )
+    version(
+        "3.8.1",
+        sha256="02e1f00e644426eb15eb08c735cf9c8ae692392f35c2cfe4f7474e1ab91575dc",
+        url="https://github.com/sfilippone/psblas3/archive/refs/tags/v3.8.1.tar.gz",
+    )
+    version(
+        "3.8.0-2",
+        sha256="86a76bb0987edddd4c10c810d7f18e13742aadc66ac14ad3679669809c1184fa",
+        url="https://github.com/sfilippone/psblas3/archive/refs/tags/v3.8.0-2.tar.gz",
+    )
+    version(
+        "3.8.0-1",
+        sha256="dca593010f258af669eb40d92c4934eaaa016ef17280d4ab1208afc3d5aa13f0",
+        url="https://github.com/sfilippone/psblas3/archive/refs/tags/v3.8.0-1.tar.gz",
+        deprecated=True,
+    )
+    version(
+        "3.8.0",
+        sha256="0eaef8f5fd41313ee1925a54bda33c5e1756a6d14b679c54e8423556881875e8",
+        url="https://github.com/sfilippone/psblas3/archive/refs/tags/v3.8.0.tar.gz",
+        deprecated=True,
+    )
+    version(
+        "3.7.0.2",
+        sha256="2c4f5ae23051b689ee9186db6baf8ca77eed7c8cb430fbc665329394036cc7f7",
+        url="https://github.com/sfilippone/psblas3/archive/refs/tags/v3.7.0-2.tar.gz",
```

Comment:
I recommend adding the `url` property but you could, alternatively, correct this argument:

```suggestion
        url="https://github.com/sfilippone/psblas3/archive/refs/tags/v3.7.0.2.tar.gz",
```